src/Types/comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
+  postId: number,
```

Comment:
Replace commas with semicolons for property definitions in the interface. Apply for other cases as well